app/assets/javascripts/discourse/app/templates/review-index.hbs

```diff
@@ -1,3 +1,27 @@
+{{#if this.displayUnknownReviewableTypesWarning}}
+  <div class="alert alert-info unknown-reviewables">
+    <span class="text">{{i18n "review.unknown.title"}}</span>
+    <ul>
+      {{#each this.unknownReviewableTypes as |type|}}
+        <li>{{type}}</li>
+      {{/each}}
+    </ul>
+    <span class="text">{{htmlSafe
+        (i18n
+          "review.unknown.instruction"
+          url="https://meta.discourse.org/t/types-of-review-items-created-by-plugins/350179"
+        )
+      }}</span>
+    <div class="unknown-reviewables__options">
+      <DButton
```

Comment:
Perhaps we could also have a button that takes them to the plugin list, so they could re-enable one quickly if needed?